nix/build/zcash-overlay.nix

```diff
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: 2022 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+self: super:
+with self.nixpkgs;
+let
+  zcash = callPackage ./zcash.nix { };
+
+  inject-zcash = oa: {
+    nativeBuildInputs = oa.nativeBuildInputs ++ [ makeWrapper ];
+    postFixup = ''
+      for bin in $(find $out/bin -not -name '*.sh' -type f -executable); do
+        wrapProgram "$bin" --prefix XDG_DATA_DIRS : ${zcash}
+      done
+    '';
+  };
+
+  relevant-packages =
+    "tezos-(accuser|node|codec|admin-client|baker|endorser|signer).*";
```

Comment:
Is there a reason not all packages that previously seemed to use `zcash` (e.g. `tezos-client`) are listed here?

Reply:
oh, good catch, indeed, seems like `tezos-client` is missing